app/assets/stylesheets/forever_style_guide/modules/_promo-banner.scss

```diff
@@ -0,0 +1,63 @@
+$promo-color: #cc5959;
+$navbar-height: 50px;
```

Comment:
it strikes me as a bit weird that this value isn't coming from the navbar component?  Also that navbar's height isn't actually set by it.